.github/workflows/unit.yml

```diff
@@ -0,0 +1,57 @@
+on: [push, pull_request]
+name: Unit test
+jobs:
+    tests:
+        name: PHP ${{ matrix.php }} / SF ^${{ matrix.symfony }}
+        runs-on: ubuntu-latest
+        strategy:
+            fail-fast: false
+            matrix:
+                php: [7.4]
+                symfony: [4.4.*]
+#                php: [7.4, 8.0, 8.1]
+#                symfony: [4.4.*, 5.4.*, 6.0.*]
+#                exclude:
+#                    - php: 7.4
+#                      symfony: 6.0.*
+
+        services:
+            rabbitmq:
+                # 3.8.10 and later versions has enabled TLSv1.3 by default which is not supported by PHP images
+                # TLSv1.3 can be disabled by using configuration file, but not environment variables
+                image: rabbitmq:3.8.9-management
+                ports:
+                    - 5671:5671
+                    - 5672:5672
+                    - 15671:15671
+                    - 15672:15672
+                env:
+                    RABBITMQ_DEFAULT_VHOST: swarrot
+
+        steps:
+            - name: Checkout
+              uses: actions/checkout@v2
+
+            - name: Setup PHP
+              uses: shivammathur/setup-php@2.16.0
```

Comment:
why pinning an exact version here and not for `actions/checkout` ?
Also, do you plan to use dependabot to maintain those pinned versions ?

Reply:
To be honest, I'm discovering github action and it's mainly copy/paste from other config I found. I removed these pinned versions.
Thank you for the heads-up